### DISCUSSION TITLE
Adding service options as properties for LocalBusiness

### DISCUSSION THREAD

**Discussion by wmwaggoner** [schemaorg/schemaorg#2963] [Ideas]
Has anyone ever suggested adding any of the attributes from Google My Business to the LocalBusiness Schema Type?

With service options for stores changing so much due to covid, it would be convenient to have those as properties. 

These are the ones I’m referring to:

Service Options
1. Curbside pickup
2. Delivery
3. In-store pickup
4. In-store shopping
5. Same-day delivery

**Comment by WeaverStever** [comment]
I think you can infer delivery (service area) with the geo tag.

https://developers.google.com/search/docs/advanced/structured-data/local-business
